src/common/utils.tsx

```diff
@@ -385,3 +385,7 @@ export const ftDecimals = (value: number | string | BigNumber, decimals: number)
 export const ftUnshiftDecimals = (value: number | string | BigNumber, decimals: number) => {
   return initBigNumber(value).shiftedBy(decimals).toString(10);
 };
+
+export function getLocaleDecimalSeparator() {
+  return (1.1).toLocaleString().substring(1, 2);
```

Comment:
Could you use the `Intl` here? I'm not sure myself, but might be appropriate https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Reply:
done ! same thing with Intl